docs/tools/cmake_in_clickhouse_generator.py

```diff
@@ -98,7 +98,9 @@ def process_folder(root_path:str, name: str) -> None:
             if f == "CMakeLists.txt" or ".cmake" in f:
                 process_file(root, f)
 
-def generate_cmake_flags_files(root_path: str) -> None:
+def generate_cmake_flags_files() -> None:
+    root_path: str = os.path.join(os.path.dirname(__file__), '..', '..')
```

Comment:
I don't understand how this change helps. Both files `build.py` and `cmake_in_clickhouse_generator.py` stored in the same directories. Why moving this line helps?